src/openforms/authentication/tests/test_signals.py

```diff
@@ -327,5 +330,6 @@ def test_set_cosigner_data(self):
                 "plugin": "digid",
                 "attribute": "bsn",
                 "value": "123456782",
+                "cosign_date": timezone.localtime().isoformat(),
```

Comment:
*please* use explicit assertions, this just repeats the implementation detail

```suggestion
                "cosign_date": "2021-11-26T16:00:00+00:00",
```